app/scripts/lib/metrics.js

```diff
@@ -339,6 +339,8 @@ define([
      */
     logScreen: function (screenName) {
       this.logEvent(this.screenToId(screenName));
+      // log the path as a separate path event
+      this.logEvent('path.' + this._window.location.pathname);
```

Comment:
> What does screenName have in it at this point?

```
[11:15:18]  <vladikoff>	stomlinson, I wonder if we can just convert screen names to paths
[11:15:34]  <vladikoff>	stomlinson, i.e screen.signup => path./signup
[11:16:13]  <stomlinson>	vladikoff: that seems reasonable.
[11:16:31]  <vladikoff>	stomlinson, so instead of adding a new event, the server will just pick those out and convert them to paths for GA
```